### ISSUE TITLE
Test failures if DateTime::Duration is not installed

### ISSUE THREAD

**Issue by eserte** [karupanerura/Time-Strptime#11]
On some systems I see the following failure:
```
#   Failed test 'No tests run for subtest "String: 2014-01-01 01:23:45 (Asia/Tokyo)"'
#   at t/03_format.t line 155.
Can't locate DateTime/Duration.pm in @INC (you may need to install the DateTime::Duration module) (@INC contains: /tmpfs/.cpan-build-cpansand/2018022312/Time-Strptime-1.02-xymK00/blib/arch /tmpfs/.cpan-build-cpansand/2018022312/Time-Strptime-1.02-xymK00/blib/lib ... .) at /home/cpansand/.cpan/build/2018022312/DateTime-TimeZone-2.17-wiCymX/blib/lib/DateTime/TimeZone/OlsonDB/Rule.pm line 9.
BEGIN failed--compilation aborted at /home/cpansand/.cpan/build/2018022312/DateTime-TimeZone-2.17-wiCymX/blib/lib/DateTime/TimeZone/OlsonDB/Rule.pm line 9.
Compilation failed in require at /home/cpansand/.cpan/build/2018022312/DateTime-TimeZone-2.17-wiCymX/blib/lib/DateTime/TimeZone/OlsonDB.pm line 9.
BEGIN failed--compilation aborted at /home/cpansand/.cpan/build/2018022312/DateTime-TimeZone-2.17-wiCymX/blib/lib/DateTime/TimeZone/OlsonDB.pm line 9.
Compilation failed in require at /home/cpansand/.cpan/build/2018022312/DateTime-TimeZone-2.17-wiCymX/blib/lib/DateTime/TimeZone/Asia/Tokyo.pm line 20.
BEGIN failed--compilation aborted at /home/cpansand/.cpan/build/2018022312/DateTime-TimeZone-2.17-wiCymX/blib/lib/DateTime/TimeZone/Asia/Tokyo.pm line 20.
Compilation failed in require at /home/cpansand/.cpan/build/2018022312/Module-Runtime-0.016-jM_lhU/blib/lib/Module/Runtime.pm line 314.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 255 just after 3.
t/03_format.t .... 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 1/3 subtests 
```

**Comment by dpnub** [comment]
I have the same problem on CentOS 7, perl 5.16.3
After installing DateTime::Duration manually I see following:

```cpanm (App::cpanminus) 1.6922 on perl 5.016003 built for x86_64-linux-thread-multi
Work directory is /root/.cpanm/work/1523017147.168260
You have make /bin/make
You have LWP 6.05
You have /bin/tar: tar (GNU tar) 1.26
Copyright (C) 2011 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by John Gilmore and Jay Fenlason.
You have /bin/unzip
Searching Time::Strptime on cpanmetadb ...
--> Working on Time::Strptime
Fetching http://www.cpan.org/authors/id/K/KA/KARUPA/Time-Strptime-1.02.tar.gz
-> OK
Unpacking Time-Strptime-1.02.tar.gz
Entering Time-Strptime-1.02
Checking configure dependencies from META.json
Checking if you have Module::Build::Tiny 0.035 ... Yes (0.039)
Checking if you have Module::Build 0.36 ... Yes (0.4005)
Configuring Time-Strptime-1.02
Running Build.PL
Creating new 'Build' script for 'Time-Strptime' version '1.02'
-> OK
Checking dependencies from MYMETA.json ...
Checking if you have Scalar::Util 0 ... Yes (1.50)
Checking if you have parent 0 ... Yes (0.225)
Checking if you have Encode 0 ... Yes (2.51)
Checking if you have Test::More 0 ... Yes (1.302135)
Checking if you have DateTime::Locale 0 ... Yes (1.18)
Checking if you have DateTime::TimeZone 0 ... Yes (2.18)
Checking if you have Time::Local 0 ... Yes (1.2300)
Checking if you have List::MoreUtils 0 ... Yes (0.33)
Building and testing Time-Strptime-1.02
cp lib/Time/Strptime/Format.pm blib/lib/Time/Strptime/Format.pm
cp lib/Time/Strptime/TimeZone.pm blib/lib/Time/Strptime/TimeZone.pm
cp lib/Time/Strptime.pm blib/lib/Time/Strptime.pm
t/00_compile.t ... ok
t/01_strptime.t .. ok

            #   Failed test at t/02_timezone.t line 52.
            #          got: '-654768001 36000'
            #     expected: '-654764401 32400'
            # Looks like you failed 1 test of 1.

        #   Failed test 'detect time_zone from option'
        #   at t/02_timezone.t line 53.

            #   Failed test at t/02_timezone.t line 57.
            #          got: '-654768001 36000'
            #     expected: '-654764401 32400'
            # Looks like you failed 1 test of 1.

        #   Failed test 'detect time_zone from env'
        #   at t/02_timezone.t line 58.

            #   Failed test at t/02_timezone.t line 63.
            #          got: '-654768001 36000'
            #     expected: '-654764401 32400'

            #   Failed test at t/02_timezone.t line 64.
            #          got: '-654768001 36000'
            #     expected: '-654764401 32400'
            # Looks like you failed 2 tests of 2.

        #   Failed test 'detect time_zone from date time string'
        #   at t/02_timezone.t line 65.
        # Looks like you failed 3 tests of 3.

    #   Failed test '1949-04-03 01:59:59'
    #   at t/02_timezone.t line 66.

            #   Failed test at t/02_timezone.t line 52.
            #          got: '-640947600 36000'
            #     expected: '-640944000 32400'
            # Looks like you failed 1 test of 1.

        #   Failed test 'detect time_zone from option'
        #   at t/02_timezone.t line 53.

            #   Failed test at t/02_timezone.t line 57.
            #          got: '-640947600 36000'
            #     expected: '-640944000 32400'
            # Looks like you failed 1 test of 1.

        #   Failed test 'detect time_zone from env'
        #   at t/02_timezone.t line 58.

            #   Failed test at t/02_timezone.t line 63.
            #          got: '-640947600 36000'
            #     expected: '-640944000 32400'

            #   Failed test at t/02_timezone.t line 64.
            #          got: '-640947600 36000'
            #     expected: '-640944000 32400'
            # Looks like you failed 2 tests of 2.

        #   Failed test 'detect time_zone from date time string'
        #   at t/02_timezone.t line 65.
        # Looks like you failed 3 tests of 3.

    #   Failed test '1949-09-10 01:00:00'
    #   at t/02_timezone.t line 66.
    # Looks like you failed 2 tests of 5.

#   Failed test 'JST-9'
#   at t/02_timezone.t line 68.

            #   Failed test at t/02_timezone.t line 52.
            #          got: '-654768001 36000'
            #     expected: '-654764401 32400'
            # Looks like you failed 1 test of 1.

        #   Failed test 'detect time_zone from option'
        #   at t/02_timezone.t line 53.

            #   Failed test at t/02_timezone.t line 57.
            #          got: '-654768001 36000'
            #     expected: '-654764401 32400'
            # Looks like you failed 1 test of 1.

        #   Failed test 'detect time_zone from env'
        #   at t/02_timezone.t line 58.

            #   Failed test at t/02_timezone.t line 63.
            #          got: '-654768001 36000'
            #     expected: '-654764401 32400'

            #   Failed test at t/02_timezone.t line 64.
            #          got: '-654768001 36000'
            #     expected: '-654764401 32400'
            # Looks like you failed 2 tests of 2.

        #   Failed test 'detect time_zone from date time string'
        #   at t/02_timezone.t line 65.
        # Looks like you failed 3 tests of 3.

    #   Failed test '1949-04-03 01:59:59'
    #   at t/02_timezone.t line 66.

            #   Failed test at t/02_timezone.t line 52.
            #          got: '-640947600 36000'
            #     expected: '-640944000 32400'
            # Looks like you failed 1 test of 1.

        #   Failed test 'detect time_zone from option'
        #   at t/02_timezone.t line 53.

            #   Failed test at t/02_timezone.t line 57.
            #          got: '-640947600 36000'
            #     expected: '-640944000 32400'
            # Looks like you failed 1 test of 1.

        #   Failed test 'detect time_zone from env'
        #   at t/02_timezone.t line 58.

            #   Failed test at t/02_timezone.t line 63.
            #          got: '-640947600 36000'
            #     expected: '-640944000 32400'

            #   Failed test at t/02_timezone.t line 64.
            #          got: '-640947600 36000'
            #     expected: '-640944000 32400'
            # Looks like you failed 2 tests of 2.

        #   Failed test 'detect time_zone from date time string'
        #   at t/02_timezone.t line 65.
        # Looks like you failed 3 tests of 3.

    #   Failed test '1949-09-10 01:00:00'
    #   at t/02_timezone.t line 66.
    # Looks like you failed 2 tests of 5.

#   Failed test 'Asia/Tokyo'
#   at t/02_timezone.t line 68.
# Looks like you failed 2 tests of 5.
t/02_timezone.t ..
Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/5 subtests
t/03_format.t .... ok

Test Summary Report
-------------------
t/02_timezone.t (Wstat: 512 Tests: 5 Failed: 2)
  Failed tests:  2-3
  Non-zero exit status: 2
Files=4, Tests=24,  6 wallclock secs ( 0.04 usr  0.01 sys +  5.48 cusr  0.68 csys =  6.21 CPU)
Result: FAIL
-> FAIL Installing Time::Strptime failed. See /root/.cpanm/work/1523017147.168260/build.log for details. Retry with --force to force install it.
```

**Comment by karupanerura** [comment]
Sorry for later. This problem is fixed on 1.03. Please check it. Thank you for reporting!
